config/routes.rb

```diff
@@ -1,14 +1,15 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  root 'components#index'
+  root 'components#index', via: :all
 
   # All the Api endpoints must be under /api/v1 and must have an extension .json.
   namespace :api do
     namespace :v1 do
       resources :sessions, only: %i[index create destroy]
       resources :users, only: [:create]
       resources :rooms, only: %i[show index], param: :friendly_id
+      post '/room/:friendly_id/start', to: 'rooms#start', as: :start_session
```

Comment:
I believe it should be ```friendlyId``` after Ahmad's most recent commit, in order to keep consistency.
@farhatahmad correct me if I'm wrong

Reply:
You'd like to keep consistency between the frontend and the backend codebase?
I don't mind it but in rails by default we usually use the `snake_case` and not the `camel case`.

Reply:
Yeah we should keep it as `snake_case` in Rails but `camelCase` in React